DataFormats/EcalDigi/src/classes_def.xml

```diff
@@ -11,21 +11,30 @@
    <class name="EcalDataFrame" ClassVersion="10">
     <version ClassVersion="10" checksum="259364627"/>
    </class>
+   <class name="EcalDataFrame_Ph2" ClassVersion="-1">
+    <version ClassVersion="-1" checksum="4122679522"/>
+   </class>
    <class name="ESDataFrame" ClassVersion="10">
     <version ClassVersion="10" checksum="2345603108"/>
    </class>
    <class name="EcalTriggerPrimitiveSample" ClassVersion="10">
     <version ClassVersion="10" checksum="251958438"/>
    </class>
-   <class name="EcalEBTriggerPrimitiveSample" ClassVersion="13">
-    <version ClassVersion="13" checksum="4171161153"/>
+   <class name="EcalEBTriggerPrimitiveSample" ClassVersion="15">
+    <version ClassVersion="15" checksum="4171161153"/>
+   </class>
+   <class name="EcalEBPhase2TriggerPrimitiveSample" ClassVersion="0">
+    <version ClassVersion="0" checksum="124268305"/>
    </class>
    <class name="EcalTriggerPrimitiveDigi" ClassVersion="12">
     <version ClassVersion="12" checksum="3058718200"/>
    </class>
    <class name="EcalEBTriggerPrimitiveDigi" ClassVersion="14">
     <version ClassVersion="14" checksum="3583827889"/>
    </class>
+   <class name="EcalEBPhase2TriggerPrimitiveDigi" ClassVersion="-1">
```

Comment:
ClassVersion -1 is new: what is it intended for?

Reply:
fixed